content/content.js

```diff
@@ -135,9 +144,9 @@ function loadForemHTML(forems) {
                       }
                     </script>`;
 
-  const foremHTML = `<style>${styleCss}</style><div id="forem-sidecar" class="forem-expanded"><base target="_parent">${
+  const foremHTML = `<style>${styleCss}</style><main id="forem-sidecar" class="forem-expanded"><base target="_parent">${
```

Comment:
Replaced `<div />` for `<main />` so that we have a landmark. Fixes the a11y issue [Page must have means to bypass repeated blocks](https://dequeuniversity.com/rules/axe/3.0/bypass).